IPs/RSKIP180.md

```diff
@@ -0,0 +1,61 @@
+# Limit the RSK merged mining merkle proof
+
+
+|RSKIP          | 180 |
+| :------------ |:-------------|
+|**Title**      |Limit the RSK merged mining merkle proof|
+|**Created**    |15-JUL-2021 |
+|**Author**     |VK |
+|**Purpose**    |Sec |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+
+# **Abstract**
+
+In this RSKIP we propose to limit the size of the bitcoin merged mining merkle proof field in a RSK block header.
+This check should happen during the block validation step.
+
+## Motivation
+
+The RSKj merge mining is compatible with the bitcoin protocol. This is: a miner can mine both bitcoin and RSK simultaneously.
```

Comment:
The RSKj merge mining -> RSK merge mining is ..
(let us use network instead of node implementation)